en/docs/api-analytics/samples/publishing-analytics-events-to-external-systems.md

```diff
@@ -0,0 +1,255 @@
+---
+title: Publishing Analytics Events to External Systems - API Manager Documentation 4.0.0
+---
+
+# Publishing Analytics Events to External Systems
+
+## Introduction
+
+It is possible to log and publish analytics events to external systems. This will guide you on how to create a sample to log analytics events and to publish those logged events to an external system. For the demonstration purpose we have selected ELK as the external system.
```

Comment:
```suggestion
Instead of publishing analytics events to the cloud, It is also possible to log the same events and publish them to external systems. This guide will explain the steps required to do it. For demonstration purposes, we have selected ELK as the external system.
```

Reply:
Fixed with ea12a8a83344ae33c5b9070693ff903eccd85600